src/libraries/Microsoft.Extensions.Logging.EventSource/ref/Microsoft.Extensions.Logging.EventSource.csproj

```diff
@@ -1,13 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
What was the purpose of the ns2.1 build?

Reply:
It was my mistake. 

I found this commit where in it I had added ns2.1:
https://github.com/dotnet/runtime/commit/473d6f9de26db5f8442bb71c9744c01de58f7d09#diff-40fd9ad4ac2ebae28ac20317489f6124